### ISSUE TITLE
Version 8: Warn users in 2D mode trying to view 3D datasets

### ISSUE THREAD

**Issue by tephenavies** [TerriaJS/terriajs#4575]
Related to #3486?

**Comment by soyarsauce** [comment]
1. go to http://ci.terria.io/master/
2. switch map to 2d
3. add data
4. aero3Dpro -> Brisbane 3D city model (aero3Dpro)
5. observe
```
Not supported in 2D

"Brisbane 3D city model (aero3Dpro)" cannot be show in the 2D view. Switch to 3D and try again.
```

do the same with http://ci.terria.io/next/, observe no 3d error modal

i first noted via https://nsw.digitaltwin.terria.io/ too when loading 3d tiles

**Comment by zoran995** [comment]
Along with 3d error modal, it might be nice to add warning box to catalog item in the workbench that it can't be viewed in 2D when 2D view is active. This might require the implementation of the listener for change between 2D and 3D viewer, it's not ported as far as I know.

**Comment by nf-s** [comment]
I'm going to mark this as closed due to merging #4879
